miden-lib/asm/kernels/transaction/lib/memory.masm

```diff
@@ -1,4 +1,4 @@
-use.std::crypto::hashes::native
+use.std::crypto::hashes::rpo
```

Comment:
Is this import here needed? It seems like nothing else in this file got changed.